backend/routes/getMe_route.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+import { authToken } from '../utils/AuthenticateToken';
+import User from '../models/user';
+
+const router = express.Router();
+
+// fetch and return the currently authenticated user's information
+router.get('/me', authToken,async (req, res) => {
+    try{
+        await User.findById(req.user._id).select("-password");
```

Comment:
I think this will always return a "user authenticated" response even if the user doesn't exist, cuz the return statement is not checking for the existence of the user, it's best to save the value returned by findById in a variable and check if it's null, then return the appropriate response

Reply:
If the user doesn't exist, the function won't go into the catch statement I believe since mongo still returns a null value instead of throwing an error